Dockerfile

```diff
@@ -5,6 +5,7 @@ MAINTAINER "Code Climate <hello@codeclimate.com>"
 USER root
 
 RUN apk update && \
+    apk add --no-cache bash && \
```

Comment:
Couldn't get the image to build or CI to pass without adding this.

Reply:
Presumably because the base image used to have `bash` installed and probably got updated to only have `sh` (which is more normal for alpine images).

I don't think we really *need* bash, and we could probably rewrite `install-pmd.sh` to use pure POSIX shell, but it's probably not worth it and this is fine.

Reply:
Would adding `bash` to the next line without the `--no-cache` option work just as well?